.changeset/pretty-buckets-develop.md

```diff
@@ -0,0 +1,5 @@
+---
+'@apollo/server': minor
+---
+
+Add `hideSchemaDetailsFromClientErrors` option to ApolloServer to allow hiding 'did you mean' suggestions from validation errors
```

Comment:
This will become a public changelog entry. I like to make sure these have all the information someone needs to know whether it's interesting to them or not and how to use it.

Can you elaborate on the problem this solves and provide a code snippet for enabling it?

Reply:
Got it! Updated with more info!